### DISCUSSION TITLE
v1.4.0-preview2

### DISCUSSION THREAD

**Discussion by duncanmacmichael** [microsoft/WindowsAppSDK#3803] [Announcements]
## 1.4 Preview 2 (1.4.230811000-preview2) 🔃

The second preview of WinAppSDK 1.4 is now available! Check out the [release notes](https://aka.ms/windowsappsdk/1.4/1.4-preview2/release-notes) for more information on the new & updated features and known issues.

### Updating to this version
The 1.4-preview2 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.4.230811000-preview2

### Providing feedback
As we introduce new & upcoming features in our preview releases, we appreciate all of your feedback as we work towards a stable release:
- For WinUI 3 feedback, please open an issue on the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/new/choose).
- For feedback regarding all other Windows App SDK components, please open an issue here, on the [Windows App SDK repo](https://github.com/microsoft/WindowsAppSDK/issues/new/choose).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.4.0-preview2'>v1.4.0-preview2</a>.</em>

**Comment by daltzctr** [comment]
Where is the known issues and bug fixes? I only see feature additions in the release notes.

**Reply by duncanmacmichael** [reply]
Hi @daltzctr, thanks for the question. There aren't any known issues with this release. As for bug fixes, we're compiling a list that we'll share along with the upcoming 1.4 stable release. Thanks for your patience!

**Reply by ghost1372** [reply]
> Hi @daltzctr, thanks for the question. There aren't any known issues with this release. As for bug fixes, we're compiling a list that we'll share along with the upcoming 1.4 stable release. Thanks for your patience!

I reported 2 issues

**Comment by DarranRowe** [comment]
I noticed that the documentation for 1.4 Preview 2 went live, but it seems to be missing the documentation for the new features.

**Reply by Karl-Bridge-Microsoft** [reply]
Hi @DarranRowe - yes, we had a publishing issue with the docs. I believe this is now resolved. Please let me know if this is not the case.

**Reply by DarranRowe** [reply]
@Karl-Bridge-Microsoft
The WinRT documentation exists, but it is still a little sparse. However, I assume that this will improve by the time 1.4 releases.
Unfortunately, the [Win32 API](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/win32/) doesn't seem to have been updated to include the changes. At the time of writing, it is missing Microsoft.UI.InputCursor.Interop.h, Microsoft.UI.Input.InputPretranslateSource.Interop.h and Microsoft.UI.Dispatching.Interop.h. Is also hasn't removed Microsoft.UI.Xaml.Hosting.DesktopWindowXamlSource.h, or at the very least made note of the fact that it was removed.

**Comment by Karl-Bridge-Microsoft** [comment]
@DarranRowe - Preview docs are typically stubs only. Typically no authored content until the Stable release. I'll have a look at the Win32 docs as soon as I can.
